test/jdk/java/lang/ScopedValue/ScopedValueAPI.java

```diff
@@ -416,13 +413,15 @@ void testNullPointerException() {
         assertThrows(NullPointerException.class, () -> ScopedValue.where(null, "duke").call(() -> ""));
         assertThrows(NullPointerException.class, () -> ScopedValue.where(name, "duke").call(null));
 
+        assertThrows(NullPointerException.class, () -> name.orElse(null));
         assertThrows(NullPointerException.class, () -> name.orElseThrow(null));
 
         var carrier = ScopedValue.where(name, "duke");
         assertThrows(NullPointerException.class, () -> carrier.where(null, "duke"));
         assertThrows(NullPointerException.class, () -> carrier.get((ScopedValue<?>)null));
         assertThrows(NullPointerException.class, () -> carrier.run(null));
         assertThrows(NullPointerException.class, () -> carrier.call(null));
+        assertThrows(NullPointerException.class, () -> carrier.run(() -> name.orElse(null)));
```

Comment:
Thanks for brining in the update to ScopedValueAPI test. The `@enablePreview` can be removed from the test description of this test, and the others in this directory that don't use STS.